Gemfile

```diff
@@ -8,7 +8,7 @@ group :sqlite do
 end
 
 group :postgresql do
-  gem "pg", "~> 1.1.4", platform: :ruby
+  gem "pg", "~> 1.2.0", platform: :ruby
```

Comment:
As the commit message says, log output was too long to see anything. Example: https://travis-ci.org/github/mixonic/ranked-model/jobs/731909186

`pg` 1.2.x fixes this, see https://github.com/ged/ruby-pg/blob/master/History.rdoc#v120-2019-12-20-michael-granger-gedfaeriemudorg-